pygen/pygen_src/isa/riscv_instr.py

```diff
@@ -9,23 +9,25 @@
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-"""
 
+"""
 
 from collections import defaultdict
-from pygen_src.riscv_instr_gen_config import riscv_instr_gen_config
-from pygen_src.riscv_instr_pkg import riscv_instr_group_t
+from pygen_src.riscv_instr_pkg import pkg_ins, riscv_reg_t
 from pygen_src.isa import rv32i_instr  # NOQA
+from pygen_src.target.rv32i import riscv_core_setting as rcs
 import random
 from bitstring import BitArray
 import logging
 from copy import copy
 import sys
 import vsc
```

Comment:
very small nit: can all of the `from...import...` statements be moved under the `import ...` statements?

Reply:
Is this fixed @aneels3 ?

Reply:
No @hodjat91,
As we run the base_test from the subdirectory of pygen_src. I have used sys module for the relative path.
I am not sure how can I move it all directly into an import statement.
Can I fix this later?

Reply:
Hi @aneels3 ,
Don't overthink it :-) I believe what Udi meant was just to list all of `import A` statements first (on top of the file, after any module comments and docstrings) and then start listing all `from B import C` statements. it is generally a good style to import modules in the following orders: Standard library imports (Python built-in modules), then related third party imports (modules that are installed and do not belong to the current application), and finally local application imports (modules that belong to the current application).

Reply:
Thanks, @hodjat91 
Got it now.
I will fix it ASAP.